packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -128,20 +145,44 @@ interface FluidDataStoreMessage {
  * being staged on IFluidParentContext can be added here as well, likely with optionality removed,
  * to ease interactions within this package.
  */
-export interface IFluidParentContextPrivate
-	extends Omit<IFluidParentContext, "isReadOnly" | "minVersionForCollab"> {
+export interface IFluidParentContextPrivate extends IFluidParentContext {
 	readonly isReadOnly: () => boolean;
-
-	/**
-	 * {@inheritdoc IFluidParentContext.minVersionForCollab}
-	 */
 	readonly minVersionForCollab: MinimumVersionForCollab;
 }
 
 /**
- * Creates a shallow wrapper of {@link IFluidParentContext}. The wrapper can then have its methods overwritten as needed
+ * Kin of {@link @fluidframework/runtime-definitions#IFluidParentContext} with alternate
+ * submitMessage and submitSignal methods.
  */
-export function wrapContext(context: IFluidParentContextPrivate): IFluidParentContextPrivate {
+export interface IFluidRootParentContextPrivate
+	extends Omit<IFluidParentContextPrivate, "submitMessage" | "submitSignal"> {
+	readonly submitMessage: (
```

Comment:
Nit: probably worth documenting the method variants here, since intellisense won't (or at least shouldn't) pull them from `IFluidParentContextPrivate`.